app/javascript/styles/_brand.scss

```
@@ -15,6 +15,11 @@
   }
 }
 
+.dropdown_scroll {
```

Selectors are typically hyphen-delimited in SCP.

```suggestion
.dropdown-scroll {
```